### DISCUSSION TITLE
Increase file size requirement

### DISCUSSION THREAD

**Discussion by LotharMadsson** [epoupon/fileshelter#80] [General]
Hey, great tool so far!

How can I change the max file size requirement that is set default to **100MB**?

Thanks!

**Comment by epoupon** [comment]
Hello,

You have to modify:
https://github.com/epoupon/fileshelter/blob/aa6ffc4ac36dec848bc880857b31d59d4ed7ec45/conf/fileshelter.conf#L20

If you use a reverse proxy, you also have to modify:
https://github.com/epoupon/fileshelter/blob/aa6ffc4ac36dec848bc880857b31d59d4ed7ec45/INSTALL.md?plain=1#L132

**Comment by andreanapol** [comment]
Hi, i cannot start container correctly with mounting custom conf  file:
 
`docker run -d --name=fileshelter --user 0:0 -p 5091:5091 -v /var/fileshelter:/var/fileshelter --restart unless-stopped --mount type=bind,source=/etc/fileshelter.conf,target=/etc/fileshelter.conf epoupon/fileshelter
`
The only one customization is set limit to 5000

**Comment by epoupon** [comment]
Hello!
What is your error?

**Comment by andreanapol** [comment]
no error, the container started with no port exposed.
I had set the size limit at 5000. Putting it instead at 900 it starts
correctly. is there a limit?


Il Ven 21 Lug 2023, 17:35 Emeric POUPON ***@***.***> ha
scritto:

**Comment by DenSnow3688** [comment]
Подскажите пожалуйста как изменить максимальный размер файла установленный по умолчанию. Мой fileshelter установлен из пакетов Debian. В файле /etc/fileshelter.conf указал "# Maximum total share size, in megabytes
max-share-size = 40000;" но независимо от "behind-reverse-proxy = false;" или "behind-reverse-proxy = true" файлы размером более 100мб не загружаются. Что я делаю не так?!
